src/main/java/com/exactpro/th2/sailfish/utils/factory/MessageFactoryProxy.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.exactpro.th2.sailfish.utils.factory;
+
+import javax.annotation.Nonnull;
+
+import com.exactpro.sf.common.impl.messages.DefaultMessageFactory;
+import com.exactpro.sf.common.messages.IMessage;
+import com.exactpro.sf.common.messages.IMessageFactory;
+import com.exactpro.sf.common.messages.structures.IDictionaryStructure;
+import com.exactpro.sf.configuration.suri.SailfishURI;
+import com.exactpro.sf.configuration.suri.SailfishURIException;
+import com.exactpro.sf.externalapi.IMessageFactoryProxy;
+import com.exactpro.sf.externalapi.impl.StrictMessageFactoryWrapper;
+
+import static java.util.Objects.requireNonNull;
+
+public class MessageFactoryProxy implements IMessageFactoryProxy {
```

Comment:
Why we created this class?

Reply:
It was created in common V3 as a default implementation of IMessageFactoryProxy to reuse it in other components. But this change was not merged into the common V2. Now I merge this change along with the changes for TS-442